src/style.css

```
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.search-small,
+.search-big {
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.search-big::before,
+.search-small::before {
+  content: "";
+  background: url("images/Search.svg") no-repeat center;
+  position: absolute;
+}
+
+.search-big::before {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.bigest {
+  font-family: "Avenir-Book", Arial, sans-serif;
+  font-size: 16px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Зроби відступи між різними групами CSS правил. Так твій код буде легше читати і підтримувати.
```suggestion
  height: 70px;
  width: 100%;
  padding-left: 62px;
  
  font-family: "Avenir-Book", Arial, sans-serif;
  font-size: 16px;
  
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```